src/gsi/read_obs.F90

```diff
@@ -1449,8 +1461,14 @@ subroutine read_obs(ndata,mype)
                    call read_prepbufr(nread,npuse,nouse,infile,obstype,lunout,twind,sis,&
                         prsl_full,nobs_sub1(1,i),read_rec(i))
                    string='READ_PREPBUFR'
-
                 endif
+
+             else if (obstype == 'gnssrspd' .and. index(infile,'gnssrwndbufr') /=0 ) then
+                write(6,*) " Before read_gnssrspd: ndat=",ndat," nread=",nread," npuse=",npuse," nouse=",nouse
```

Comment:
This write statement seems to be leftover from initial code development or debugging.  Please remove this line.

Reply:
Removed.